### PR TITLE
Retries count and retry pause is configurable for all Asset Ingestors

### PR DESCRIPTION
**Pull request by cylinder-y**
new configuration field was added for FAM tasks:
-retry count
-retry pause

### CODE REVIEW

**Comment by badvision** [comment]
I'm not sure that having the timeout replaced with retry_pause is a good idea.  timeout is more of a connection timeout interval and that could be several seconds.  Retry pause in the FAM actions class is something that is usually in milliseconds.

File: bundle/src/main/java/com/adobe/acs/commons/mcp/impl/processes/asset/FileAssetIngestor.java [hunk 1]
```diff
@@ -333,7 +324,7 @@ private ChannelSftp openChannel() throws JSchException {
 
                 com.jcraft.jsch.Session session = jsch.getSession(username, uri.getHost(), port);
                 session.setConfig("StrictHostKeyChecking", "no");
-                session.setTimeout(timeout);
```
Comment (badvision) [inline bundle/src/main/java/com/adobe/acs/commons/mcp/impl/processes/asset/FileAssetIngestor.java]: I think that timeout is not the same as retry pause

File: bundle/src/main/java/com/adobe/acs/commons/mcp/impl/processes/asset/UrlAssetImport.java [hunk 2]
```diff
@@ -140,11 +131,11 @@ public void init() throws RepositoryException {
             HttpClientBuilder clientBuilder = httpFactory.newBuilder();
             clientBuilder.setDefaultSocketConfig(
                     SocketConfig.custom()
-                            .setSoTimeout(timeout)
```
Comment (badvision) [inline bundle/src/main/java/com/adobe/acs/commons/mcp/impl/processes/asset/UrlAssetImport.java]: I think that timeout should be its own setting

**Comment by cylinder-y** [comment]
Hi @badvision,
Thanks a lot for reviewing, I have rolled back timeout field
--Yaraslau

**Pull request merged**